docs/4.1/aws_oss_guide.md

```diff
@@ -124,6 +124,67 @@ proxy and provides the TCP connections needed for Teleport proxy SSH connections
 IAM is the recommended tool for creating service access. This guide will follow the 
 best practice of principle of least privilege (PoLP). 
 
+#### IAM for Amazon S3
+
+In order to grant an IAM user in your AWS account access to one of your buckets, `example.s3.bucket` you will need to grant the following permissions: `s3:ListBucket`, `s3:ListBucketVersions`, `s3:PutObject`, `s3:GetObject`, `s3:GetObjectVersion`
+
+An example policy is shown below:
+
+```
+{
+   "Version": "2012-10-17",
+   "Statement": [
+     {
+       "Effect": "Allow",
+       "Action": [
+         "s3:ListBucket",
+         "s3:ListBucketVersions"
+        ],
+       "Resource": ["arn:aws:s3:::example.s3.bucket"]
+     },
+     {
+       "Effect": "Allow",
+       "Action": [
+         "s3:PutObject",
+         "s3:GetObject",
+         "s3:GetObjectVersion"
+       ],
+       "Resource": ["arn:aws:s3:::example.s3.bucket/*"]
+     }
+   ]
+ }
+```
+!!! note "Note":
+    `example.s3.bucket` will need to be replaced with your bucket name. 
+
+#### IAM for DynamoDB
+
+In order to grant an IAM user access to DynamoDB make sure that the IAM role assigned to Teleport is configured with proper permissions. 
+
+An example policy is shown below:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AllAPIActionsOnTeleportAuth",
+            "Effect": "Allow",
+            "Action": "dynamodb:*",
```

Comment:
^ As per https://github.com/gravitational/teleport/issues/3085 we should likely reduce the `*` scope for selecting dynamobb and instead call out the specific name. 

( This looks like It was also merged in the last update )